terasoluna-tourreservation-web/src/main/resources/META-INF/spring/application.properties

```diff
@@ -2,3 +2,6 @@
 customer.initialBirthYear=1980
 customer.initialBirthMonth=1
 customer.initialBirthDay=1
+
+reservation.report.pdf.name=reservationReport.pdf
+reservation.report.pdf.path=classpath:reports/reservationReport.pdf
```

Comment:
- [ ] Good to adjust property name rule.
As `reservation.reportPdfName`, `reservation.reportPdfUrl`.

Reply:
e7b16d860d3fe56c4210c7bbe530c51cdd25f089

I adjust prooperty name rule in above commit .